src/main/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbTemplate.java

```diff
@@ -170,15 +166,10 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
         final FeedOptions feedOptions = new FeedOptions();
         feedOptions.setEnableCrossPartitionQuery(true);
 
-        SqlQuerySpec sqlQuerySpec = new SqlQuerySpec("SELECT * FROM root c");
-        if (partitionKeyFieldName != null && !partitionKeyFieldName.isEmpty()) {
-            sqlQuerySpec = new SqlQuerySpec("SELECT * FROM root c WHERE c." + partitionKeyFieldName + "=@partition",
-                    new SqlParameterCollection(new SqlParameter("@partition", partitionKeyFieldValue)));
-            feedOptions.setPartitionKey(new PartitionKey(partitionKeyFieldValue));
-        }
+        final SqlQuerySpec sqlQuerySpec = new SqlQuerySpec("SELECT * FROM root c");
 
         final List<Document> results = documentDbFactory.getDocumentClient()
-                .queryDocuments(collections.get(0).getSelfLink(), sqlQuerySpec, feedOptions, partitionKeyFieldName)
+                .queryDocuments(collections.get(0).getSelfLink(), sqlQuerySpec, feedOptions)
```

Comment:
set feedOption always no matter collection partitioned or not?

Reply:
Thanks, for the findAll, the enableCrossRegion will be enabled as no PartitionKey value is provided, but I have made change for other 2 methods in this [commit](https://github.com/Microsoft/spring-data-documentdb/pull/37/commits/25b4a2eaf311ea23f868ede9b86b41484a25dab8)